library/general/src/lib/y2user/reader/getent.rb

```diff
@@ -0,0 +1,117 @@
+require "yast2/execute"
+require "date"
+
+require "y2user/group"
+require "y2user/user"
+require "y2user/password"
+
+module Y2User
+  module Reader
```

Comment:
Shouldn't this be `Y2User::Readers` (plural) in the same way we have `Y2Network::ConfigWriters`, `Y2Network::Dialogs`, `Y2Storage::Callbacks`, Y2Storage::Clients` and many more?

Reply:
Maybe yes, I always think about it and never decide in just one way. So lets unify it.